ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -369,25 +586,82 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
+- as: gcp-ipi-sdn-f28
+  cron: 33 21 21 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-sdn
 - as: gcp-upi-f28
-  cron: 55 10 19 * *
+  cron: 55 7 1 * *
   steps:
     cluster_profile: gcp-qe
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-upi
-- as: nutanix-ipi-fips-f14
+- as: gcp-upi-private-xpn-ingress-glb-f28
+  cron: 39 12 18 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-ingress-glb
+- as: gcp-upi-private-xpn-sdn-f28
```

Comment:
the sdn is the default networkType in 4.11, so I think this one is duplicated with `gcp-upi-private-xpn-ingress-glb` from networkType coverage view.

Reply:
We only have two gcp-xpn jobs: `gcp-upi-private-xpn-ingress-glb-f28` and `gcp-upi-private-xpn-sdn-f28`, so I think it's better to keep this one.

Reply:
ok， got it